describealign.py

```diff
@@ -99,15 +101,14 @@
 import multiprocessing
 import platform
 
+default_output_dir = platformdirs.user_videos_path() / 'videos_with_ad'
+default_alignment_dir = platformdirs.user_data_path('describealign') / 'alignment_plots'
+
 IS_RUNNING_WINDOWS = platform.system() == 'Windows'
 if IS_RUNNING_WINDOWS:
   import PySimpleGUIWx as sg
-  default_output_dir = 'videos_with_ad'
-  default_alignment_dir = 'alignment_plots'
```

Comment:
Are these broken?  I can't say I was ever happy with the Mac implementation I slapped together here (originally to fix issues with the binary), but I think we should have a good reason for changing the default output directories.

Reply:
Mainly this was a consistency change to keep the dirs the same even if you reinstall/redownload the build (and possibly run it from a different dir). I can definitely change these back if you'd prefer though

Reply:
Good point, I'll revert this for now, the $HOME based paths on mac+linux do solve the issues